interfaces/PA-App/src/app/personal-components/select-language/select-language.component.ts

```diff
@@ -6,6 +6,9 @@ import { PaDataService } from 'src/app/services/padata.service';
 import { TranslateService } from '@ngx-translate/core';
 import { ConversationService } from 'src/app/services/conversation.service';
 
+import { PopoverController } from '@ionic/angular';
```

Comment:
This should not be necessary when using the "MoreInfoButtonComponent" via the "SharedModule"..